extensions/oss-browser/src/components/folder.tsx

```diff
@@ -0,0 +1,375 @@
+        updateBookmarksState(Object.keys(bookmarks).filter(filterBookmark).map(getFolderByLocalKey));
+        return;
+      }
+      updateLoadingState(true);
+      await searchByMark(props.path, "");
+    } catch (error) {
+      updateErrState(true);
+    } finally {
+      updateLoadingState(false);
+    }
+  }
+
+  async function onSearch(text: string) {
+    try {
+      updateLoadingState(true);
+      await searchByMark(`${props.path}${text}`, "");
+    } catch (error) {
+      updateErrState(true);
+    } finally {
+      updateLoadingState(false);
+    }
+  }
+
+  async function loadMore() {
+    if (isLoadingState) {
+      return;
+    }
+    try {
+      updateLoadingState(true);
+      await searchByMark(queryState, markerState);
+    } catch (error) {
+      updateErrState(true);
+    } finally {
+      updateLoadingState(false);
+    }
+  }
+
+  async function searchByMark(query: string, marker: string) {
+    updateQueryState(query);
+    const res = await listPage(query, marker);
+    updateObjectsState((oldObjects) => {
+      const searchObjects = res.objects
+        .filter((o) => o.size != 0)
+        .map((o) => {
+          return {
+            name: o.name,
+            url: o.url,
+            lastModified: o.lastModified,
+            type: o.type,
+            size: o.size,
+          };
+        });
+      if (marker) {
+        oldObjects.push(...searchObjects);
+        return oldObjects;
+      }
+      return searchObjects;
+    });
+    updateFoldersState((oldFolders) => {
+      const searchFolders = res.prefixes || [];
+      if (marker) {
+        oldFolders.push(...(searchFolders || []));
+        return oldFolders;
+      }
+      return searchFolders;
+    });
+    updateMarkerState(res.nextMarker);
+  }
+
+  async function onBucketChange(bucket: string) {
+    try {
+      const bucketInfo = bucketsState.find((b) => b.name === bucket);
+      if (bucketInfo && newOssClient(bucketInfo.name, bucketInfo.region)) {
+        updateLoadingState(true);
+        const [bookmarks] = await Promise.all([LocalStorage.allItems(), searchByMark(props.path, "")]);
+        updateBookmarksState(Object.keys(bookmarks).filter(filterBookmark).map(getFolderByLocalKey));
+      }
+    } catch (error) {
+      updateErrState(true);
+    } finally {
+      updateLoadingState(false);
+    }
+  }
+
+  if (errState) {
+    return <ErrView />;
+  }
+  return (
+    <List
+      isLoading={isLoadingState}
+      throttle={true}
+      searchBarPlaceholder={`Search in $Root/${props.path}`}
+      onSearchTextChange={onSearch}
+      searchBarAccessory={
+        !props.path ? <BucketDropdown buckets={bucketsState} onBucketChange={onBucketChange} /> : null
+      }
+      actions={
+        <ActionPanel>
+          <CommonActions currentFolder={props.path} refresh={refresh} marks={bookmarksState} />
+        </ActionPanel>
+      }
+    >
+      <List.EmptyView title="No data" />
```

Comment:
```suggestion
      <List.EmptyView title="No Data" />
```
You could also include a logo in 64x64 px size 🙂

Reply:
A logo? Do you mean the icon when the list is empty? I think the default icon is ok 😅